scripts/healthz

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+currentDate=`date +%s%N | cut -b1-13`
+fileDate=$(head -n 1 /tmp/calculation-service-healthz.txt)
+difference=`expr $currentDate - $fileDate`
+
+if [ $difference -gt 10000 ]; then
```

Comment:
is there the danger of false positives if the time is set to the exactly the refresh time, i.e. a few ms could potentially be lost between file writes.

Reply:
This is a typo, should have been 30000 to give a safety buffer, will fix.

Reply:
maybe the issue would have been caught by some tests...

Reply:
Fixed